dist/main.css

```diff
@@ -0,0 +1,94 @@
+.jumbotron {
+  padding-top: 48px;
+  padding-bottom: 48px;
+}
+
+body {
+  margin: 0px;
+}
+
+#app {
+  background-color: #DCDCDC;
+  text-align: center;
+  padding: 0;
+  display: block;
+  margin: 0;
+}
+
+label {
+  float: left;
+}
+
+button {
+  float: left;
+  background-color: red;
+  border-radius: 5px;
+  color: white;
+  font-family: Arial;
+  font-size: 16px;
+  padding: 10px 46px;
+  vertical-align: middle;
+}
+
+a {
+  color: red;
+  font-family: helvetica;
+  text-decoration: none;
+  display: inline-block;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.btn-link {
+  padding-top: 10px;
+  padding-right: 150px;
+  vertical-align: middle;
+}
+
+input {
+  width: 400px;
+  border: 1px solid red;
+  border-radius: 5px;
+  height: 30px;
+  font-size: 16px;
+  padding-left: 5px;
+}
+
+h2 {
+  margin-top: 0;
+  font-size: 30px;
+}
+
+.mainSector {
+  margin-top: 0;
+  background-color: #DCDCDC;
+  height: 500px;
+  text-align: center;
+}
+
+.help-block {
+  color: red;
+  font-size: large;
+}
+
+.form-group {
+  display: block;
+  width: 400px;
+  height: 50px;
+  margin: 0 auto;
+  font-size: 20px;
+  padding-bottom: 25px;
+  margin-bottom: 15px;
+}
+
+.text-center {
+  text-align: center;
+}
+
+img {
+  /*-moz-transform-origin: right top; */
+  height: 300px;
+}
+/*# sourceMappingURL=main.css.map */
```

Comment:
Remove ghost code and add an empty line at the end of the file

Reply:
Thanks.